src/Presentation/Nop.Web/Infrastructure/Permissions/ProductReview/IProductReviewPermissionService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Nop.Core.Domain.Catalog;
+using Nop.Core.Domain.Customers;
+using Nop.Core.Domain.Stores;
+
+namespace Nop.Web.Infrastructure.Permissions.ProductReview
+{
+    public interface IProductReviewPermissionService

```

Comment:
i added this as a permission service, actually the name is not good, will be great if you can share a proper name im thinking also about IProductReviewPermissionHandler
-
at the moment i didnt saw a way to have some places to be able to share this code, as i cant move this to Nop.Services as here i use some caching staff, and CacheKey creator is in Nop.Web, not sure what approach is taking right now for this kind of things, but will appreciate if we can have a chat on this

Reply:
denisburlac - how about naming it as "IProductReview**Restriction**Service.cs"